FirebaseFunctions/Tests/Unit/FunctionsAPITests.swift

```diff
@@ -49,6 +49,8 @@ final class FunctionsAPITests: XCTestCase {
 
     let callableRef = Functions.functions().httpsCallable("setCourseForAlderaan")
     callableRef.timeoutInterval = 60
+    let url = URL(string: "https://localhost:8080/setCourseForAlderaan")!
+    let callableRefByURL = Functions.functions().httpsCallable(url: url)
```

Comment:
Can you also add one for the Codable API? I realize now that we don't have it for the other methods so let me know if you need a hand generating it.

Reply:
Figured it out.

Reply:
That isn't the API surface that folks use and `new` isn't a keyword in Swift so I'm not sure how it's passing if it is - I have a change locally that adds the full tests just building locally to verify